Lib/test/test_crossinterp.py

```diff
@@ -983,8 +1339,8 @@ def test_tuple(self):
 
     def test_tuples_containing_non_shareable_types(self):
         non_shareables = [
-                Exception(),
-                object(),
+                EXCEPTION,
+                OBJECT,
```

Comment:
nit: indent.  Also, `cls` is undefined at L726, 732:
```py
assert not hasattr(mod, func.__name__), (cls, getattr(mod, func.__name__))
```

Reply:
fixed